qemu/tests/cfg/unittest.cfg

```diff
@@ -1,5 +1,5 @@
 - unittest:
-    type = unittest
+    type = kvm_unit_test
```

Comment:
please rename  python file too.

Reply:
Hello @xutian it is renamed, see the commit....